content/concept-amendments.md

```diff
@@ -138,16 +141,23 @@ A transaction remains in the queue until one of the following happens:
 * It becomes invalid (for example, the [`LastLedgerSequence`](reference-transaction-format.html#lastledgersequence) causes it to expire)
 * It gets dropped because there are too many transactions in the queue with a higher transaction cost.
 
-## FlowV2 ##
+## Flow ##
 
 | Amendment ID | Status |
 |--------------|--------|
-| 5CC22CFF2864B020BD79E0E1F048F63EF3594F95E650E43B3F837EF1DF5F4B26 | Planned for voting, expected to be enabled 2016-08-24 |
+| 740352F2412A9909880C23A559FCECEDA3BE2126FED62FC7660D628A06927F11 | TBD |
 
-Replaces the payment processing engine with a more robust and efficient rewrite called the FlowV2 engine. The new version of the payment processing engine is intended to follow the same rules as the old one, but occasionally produces different results due to floating point rounding.
+Replaces the payment processing engine with a more robust and efficient rewrite called the Flow engine. The new version of the payment processing engine is intended to follow the same rules as the old one, but occasionally produces different results due to floating point rounding. This Amendment was created to replace the [FlowV2](#flowv2) amendment when a critical bug was found in the FlowV2 amendment.
 
-The FlowV2 Engine also makes it easier to improve and expand the payment engine with further Amendments.
+The Flow Engine also makes it easier to improve and expand the payment engine with further Amendments.
+
+## FlowV2 ##
+
+| Amendment ID | Status |
+|--------------|--------|
+| 5CC22CFF2864B020BD79E0E1F048F63EF3594F95E650E43B3F837EF1DF5F4B26 | Failed to hold a majority. To be removed. |
 
+This amendment was intended to replace the payment processing engine with a more robust and efficient rewrite called the FlowV2 engine. However, a critical bug was found during the voting period, so key validators vetoed the Amendment and it lost its majority. Ripple plans to remove the FlowV2 amendment in future versions of `rippled` and replace it with the [Flow](#flow) amendment.
```

Comment:
I don't know if we want to say "critical bug" as opposed to using the wording we used on the announcement: "flaw". Additionally, should we link to the commit that addresses this? If so, it's here: https://github.com/ripple/rippled/commit/b92a7d415eecb07ace2b72b6792d9dfa489c5a04